src/NuGet.Clients/NuGet.PackageManagement.UI/Models/PackageDetailControlModel.cs

```diff
@@ -206,6 +206,7 @@ public override void OnSelectedVersionChanged()
         {
             base.OnSelectedVersionChanged();
             OnPropertyChanged(nameof(IsSelectedVersionInstalled));
```

Comment:
You can remove all property changed for `IsSelectedVersionInstalled`, because as I can tell, it's no longer bound to anything.